src/Components/SampleEditor/TabNavigation/TabNavigation.tsx

```diff
@@ -46,10 +48,12 @@ export class TabNavigation extends Component<TabNavigationProps, TabNavigationSt
     const executable = FeatureToggleClient.isFeatureEnabled(featureFlags.enableEditor);
     return (
       <TabNav showClose={false}>
-        <TabNavigationAction onClick={this.props.onShowReadme}>
-          <div className="icon icon-info" style={this.props.showReadme ? { display: "inline-block", color: "white" } : { display: "inline-block" }}></div>
-        </TabNavigationAction>
-        {executable && <RunCodeButton style={{ paddingLeft: "10px", paddingRight: "10px" }} onRunStarted={this._onRunStarted} onBundleError={this._onBundleError} onRunCompleted={this._onRunCompleted} buildOnRender={false} />}
+        <div className="action-item " onClick={this.props.onShowReadme}>
```

Comment:
Should these changes go in @bentley/monaco-editor @StefanRetief. Or is this okay for now

Reply:
I'll apply the same changes in the editor and it should apply in the next update, for the time being, this should be okay.